### PR TITLE
Zephyr: fix the link of guide

### PR DESCRIPTION
**Pull request by supperthomas**
Signed-off-by: supperthomas <78900636@qq.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,11 +297,11 @@ Install required Python modules with:
 Check out the guide how to set up AutoPTS for Zephyr + nRF52 under:
 - Linux + Windows 10 virtual machine
 
-  https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/guides/bluetooth/autopts/autopts-linux.rst
+  https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/connectivity/bluetooth/autopts/autopts-linux.rst
```
Comment (MariuszSkamra) [inline README.md:300]: https://docs.zephyrproject.org/latest/connectivity/bluetooth/autopts/autopts-linux.html

File: README.md [hunk 2]
```diff
@@ -297,11 +297,11 @@ Install required Python modules with:
...
 
 - Windows 10
 
-  https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/guides/bluetooth/autopts/autopts-win10.rst
+  https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/connectivity/bluetooth/autopts/autopts-win10.rst
```
Comment (MariuszSkamra) [inline README.md:304]: https://docs.zephyrproject.org/latest/connectivity/bluetooth/autopts/autopts-win10.html

**Review by MariuszSkamra — CHANGES_REQUESTED** [review]
Nice catch, could you please use the links to the Zephyr documentation instead?

**Comment by supperthomas** [comment]
done

**Review by MariuszSkamra — APPROVED** [review]

**Pull request merged**